app/(api)/auth/user/DELETE.tsx

```diff
@@ -0,0 +1,58 @@
+'use server';
+
+import { NextRequest, NextResponse } from 'next/server';
+import { getDatabase } from '@utils/mongodb/mongoClient.mjs';
+import { HttpError, NotFoundError } from '@utils/response/Errors';
+import { ObjectId } from 'mongodb';
+
+// pass in id of user to delete
+// check if user exists in db before deleting
+/*
+ *   Removes existing user in database
+ *   @param request: {_id: string}
+ *   @returns {ok: boolean, body: {_id: string, email: string, password: string, removedCount: number}, error: string}
+ */
+const collectionName = 'users';
+
+export async function DELETE(request: NextRequest) {
```

Comment:
This isn't an API endpoint. it's just a function in datalib so it just needs to take in parameters regularly. It can still return a NextRequest though if you want. You can talk to Win about that part. I told Win you can choose whether or not to return NextRequest or JSON. I don't mind as long as you guys agree.